frontend/src/components/navigation/MobileNavigation.module.scss

```diff
@@ -0,0 +1,118 @@
+@import "../../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.mobile-menu {
+  width: 100%;
+  position: absolute;
+  left: 0;
+  // setting the z-index low to allow mobile
+  // menu to slide behind main navigation header
+  z-index: 2;
+  top: $navHeight;
+  background: $color-white;
+  flex-direction: column;
+  box-shadow: $box-shadow-lg;
+
+  &:not(is-active) {
+    animation: slide-out 0.55s cubic-bezier(0.52, -0.14, 0.31, 1.2);
+    animation-iteration-count: 1;
+    animation-fill-mode: forwards;
+  }
+  &.is-active {
+    animation: slide-in 0.75s cubic-bezier(0.52, -0.14, 0.31, 1.2);
+    animation-iteration-count: 1;
+    animation-fill-mode: forwards;
+  }
+
+  @keyframes slide-in {
+    0% {
+      transform: translateY(-100%);
+    }
+    100% {
+      transform: translateY(0);
+    }
+  }
+  @keyframes slide-out {
+    100% {
+      transform: translateY(-100%);
+    }
+  }
+}
+
+.menu-item-list {
+  @media screen and #{$mq-md} {
+    display: none;
+  }
+}
+
+.menu-item {
+  display: flex;
+
+  @include text-display-xxs;
+  padding: $spacing-lg $spacing-md;
+  font-family: $font-stack-firefox;
+  border-bottom: 2px solid #ececec;
+
+  .link {
+    display: flex;
+    flex-direction: row;
+    gap: $spacing-md;
+    font-weight: 500;
+    color: $color-grey-40;
+    cursor: pointer;
+
+    path {
+      fill: $color-purple-40;
+    }
```

Comment:
I think the `Cogwheel` was giving me issues with this - all other icons fall in line - I added `style` and set fill to `currentcolor` for `Cogwheel`. It also had a fill property within the path, removed that, now it depends on the former. Otherwise, the hardcoded color takes precedence.

Reply:
Yeah, I think `<Cogwheel>` is before I came up with the `currentColor` setup (or even had a desire to use different colours in different situations), which therefore still hardcoded a fill colour.